packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react.test.ts.snap

```diff
@@ -5475,15 +5475,14 @@ import {
   EscapeHatchProps,
   getOverrideProps,
 } from \\"@aws-amplify/ui-react/internal\\";
-import { GridCustom } from \\"./Grid\\";
+import { Grid as GridCustom, GridProps } from \\"./Grid\\";
 import ViewTest, { ViewTestProps } from \\"./ViewTest\\";
-import { GridCustomProps } from \\"./GridCustom\\";
 
 export declare type PrimitiveOverrideProps<T> = Partial<T> &
   React.DOMAttributes<HTMLDivElement>;
 export declare type AliasParentOverridesProps = {
   AliasParent?: ViewTestProps;
-  Grid?: GridCustomProps;
+  Grid?: GridProps;
```

Comment:
Is there a case where the primitive GridProps is used and a custom component GridProps?

Reply:
I believe so, that would mean the user is using both the Grid primitive and a custom Grid component as children in the same parent, which seems unlikely in my opinion.

Would it be best to alias the custom component props as well like what was done for the component? i.e. `import { GridProps as GridCustomProps } from "./Grid"`

Reply:
Yeah, that would make sense. Can you add a test case for that?

Reply:
Yeah sounds good, I'll have it in the next rev of this PR.

Reply:
Updated validation test to be more specific for parent-child components with the same name and child components with the same name in new rev.